src/utils/string.ts

```diff
@@ -4,16 +4,15 @@ import { strip0x } from '@hyperlane-xyz/utils';
 // Only allows letters and numbers
 const alphanumericRgex = /[^a-zA-Z0-9]/gi;
 export function sanitizeString(str: string) {
-  if (!str || typeof str !== 'string') return '';
```

Comment:
Can please you restore the `typeof` check here?

Reply:
Of course, but I have a question . If  the parameter type of "str" is string in typescript , does typeof str !== 'string' always evaluate to false? 
![32c1789e-9245-443f-ad4a-70d0db2d6368](https://github.com/user-attachments/assets/aeeee657-d708-400a-a024-69957150b4c9)

Reply:
Not necessarily. Typescript only does **build-time** checking. At **runtime**, it's still possible due to developer mistakes or invalid data somewhere for a non-string value to get passed into this function. That's why this defensive check is still useful :)

Reply:
I got it. I have already fixed this issue. Thanks for your patience in answering my questions.